app/models/project.rb

```diff
@@ -0,0 +1,9 @@
+class Project < ApplicationRecord
+  belongs_to :user
+  has_many(:tasks,
+           -> { order(position: :asc) },
```

Comment:
By default order used `:asc` order, so just keep
```suggestion
           -> { order(:position) },
```